### DISCUSSION TITLE
How do we find Design Systems Engineering events?

### DISCUSSION THREAD

**Discussion by GarthDB** [DesignSystemsEngineer/designsystemsengineer.github.io#39] [General]
## Introduction

The start of these discussions has really been motivated, in part, by conversations with @pfulton @nklansford @zastrow @bencallahan @5t3ph @cchaos @erunyon and others at [Front-end Design Conference 2024](https://frontenddesignconference.com/) between and in the middle of sessions. I think events are a pretty important part of building a community and bringing people together.

## The problem

How do we find the events where we would find front-end and design system content/sessions and attendees? Word of mouth is good, but does anyone have any brilliant ideas on how to avoid good events slipping through the cracks?

This problem would be further complicated by trying to keep track of things happening in areas outside of the US.

**Comment by ddamato** [comment]
Shameless plug for [ds.events](https://ds.events/) 😄

**Reply by GarthDB** [reply]
Is it 100% self-reporting by events to be listed there, or is it aggregated from additional sources?

**Reply by ddamato** [reply]
No aggregation, only user submitted events curated by a small team.

**Reply by erunyon** [reply]
@ddamato FYI, the site doesn't load content on Firefox: `Uncaught RangeError: invalid time zone in DateTimeFormat()`

**Reply by ddamato** [reply]
Thanks for reporting. Probably not a Firefox error, It's probably your location. Where are you located?

**Reply by erunyon** [reply]
@ddamato South-west Michigan USA (eastern timezone). The site loads fine on Safari and Chrome.

**Reply by ddamato** [reply]
Weird af, works on my Firefox on different machines. Probably just need to skip the timezone if it fails. Only seen this once before in the past 9 months.

**Comment by cchaos** [comment]
It's also not just about quantity but quality too. Ones that aren't purely driven to be just a plug for specific platforms/applications. Or ones that only promote the same 3-4 people in the DS space. 

I also feel way more apprehensive about spending $500 for **virtual** events. I'm just of the mindset that education and learning should be **accessible**, and these feel somewhat gate-keepy.

**Reply by GarthDB** [reply]
@cchaos additional data that matters to me as well, is who do I know that is going, not just speakers but also attendees. Front End Design Conf tried to help with that, but this community could allow for more of that, like here's an event, here's who's attending and speaking from this community. Then afterward, we could give write-ups of what we found useful and what we wish we had heard more about.

I would like to be able to include every event we can find but also provide the necessary context that would help people decide which events best help them, maybe even filter them.

**Comment by nklansford** [comment]
Our conversation revealed to me how heavy DS resources lean towards design (this is reinforced to me because I tend to look for those kinds of resources). DS communities talk a lot about design x culture x org structure but I don't hear as many people talking about how engineering processes relate to those decisions (other than just building it). What I do hear is a resounding "Involve engineering earlier!!!" 

That was a lot of sentiments. What I think I mean is:

- In my own experience, I've found it hard to have subjective conversations about engineering-related topics
- Design tends to have a lot of opinions
- Do engineers want to have open-ended conversations about process?
- If so, what events and resources line up with that? Do enough of them exist?

**Reply by nklansford** [reply]
TLDR: I don't have an answer, but rather an enthusiastic echo of the original question :)

**Reply by GarthDB** [reply]
Good call on all of this. It can be easy for people/events to fall into tech stack camps instead of collaborating on shared problems and solutions.
